to_production.txt

```diff
@@ -1,16 +1,19 @@
 # New
-ofl/suse # https://github.com/google/fonts/pull/8044
+ofl/eduauvicwantdots # https://github.com/google/fonts/pull/8020
```

Comment:
Do we need the tagging CSV push for any of these new families?

Reply:
@m4rc1e?